libraries/radwordsprocessing/getting-started.md

```diff
@@ -23,23 +23,50 @@ This tutorial will take you through the creation of a sample application that us
 
 ## Assembly References
 
-Here is a list of assemblies that contain the __RadWordsProcessing__ functionality and need to be referenced in your project:
-        
-
-* **Telerik.Windows.Documents.Core.dll**
-* **Telerik.Windows.Documents.Flow.dll**
-* **Telerik.Windows.Zip.dll**
+>The libraries support .NET 4 and later. .NET Standard-compatible binaries are available as well. The assemblies for .NET Standard don't include 'Windows' in their names. (e.g. **Telerik.Documents.Core.dll**). For more information check [**Cross-Platform Support**]({%slug radwordsprocessing-cross-platform%}) article.
 
-If you need to export documents to PDF format, you will need to refer the following assemblies:
-
-* **Telerik.Windows.Documents.Flow.FormatProviders.Pdf.dll** 
-* **Telerik.Windows.Documents.Fixed.dll**
-            
-To describe different colors, shapes and other properties, **RadWordsProcessing** depends on the listed below .NET assemblies, which you should also refer in your project:
+Here is a list of assemblies that contain the __RadWordsProcessing__ functionality and need to be referenced in your project:
 
-* **WindowsBase.dll**
-* **PresentationCore.dll**
-* **PresentationFramework.dll**
+<table>
```

Comment:
Telerik.Windows.Documents.Flow.FormatProviders.Pdf.dll and Telerik.Windows.Documents.Fixed.dll are missing

Reply:
Now added.